### ISSUE TITLE
Update master to latest GST-MFX codebase

### ISSUE THREAD

**Issue by ishmael1985** [01org/gstreamer-media-SDK#49]
The new GST-MFX 2.0 codebase is leaner, cleaner and even more feature-packed with features compared to the current 01.org master. It also is cross-platform with full hardware acceleration on both Windows and Linux. There is no reason why the current 01.org GStreamer Intel Media SDK implementation should not move to the new GST-MFX implementation as explained. Is there a timeline as to when the migration will occur?

**Comment by bengji** [comment]
I agree with this. We been using other forks instead of 1.org months now.

**Comment by ishmael1985** [comment]
@SiewHoon @rlim77 it's been many months since this issue has been opened, and it looks like there is hardly an progress in this. What is the rationale as to why the transition has been extremely slow? Is there even any intention to transition and finally bring the topic_linux_and_window branch as the new master?

**Comment by pauldotknopf** [comment]
What fork/branch should I be using these days for each platform?

Windows?

Linux?

There seriously needs to be convergence already...

**Comment by ishmael1985** [comment]
@SiewHoon @rlim77 any updates on this?
